CHANGELOG.md

```diff
@@ -40,12 +40,18 @@ All notable changes to this project will be documented in this file.
 -- IsThreeDSecureTwo (new)
 -- NoOfAuthAttempts (new)
 - Transactions.Get and GetByReceipt now return GetWebPaymentResponseModel
+- Rename PkPaymentModel to ApplePayPaymentModel
```

Comment:
Does this mean our API now exposes a field named `applePayPayment` instead of `pkPayment` ?

Reply:
No, the JudoPay API stays with pkPayment and this is what the DotNet SDK will send - I just think it's easier for merchants to understand to call this ApplePayPaymentModel in this SDK